apps/api/src/app/portfolio/calculator/roai/portfolio-calculator-btcusd.spec.ts

```diff
@@ -142,19 +142,22 @@ describe('PortfolioCalculator', () => {
         valueWithCurrencyEffect: 0
       });
 
+      /**
+       * Closing price on 2021-12-12: 50098.3
+       */
       expect(portfolioSnapshot.historicalData[1]).toEqual({
         date: '2021-12-12',
         investmentValueWithCurrencyEffect: 44558.42,
-        netPerformance: -4.46,
+        netPerformance: 5535.42, // 1 * (50098.3 - 44558.42) - 4.46 = 5535.42
```

Comment:
Why are `netPerformanceInPercentage` and `netPerformanceInPercentageWithCurrencyEffect` equal to `0` when `netPerformance` has a positive value?